docs/aerodromes/Moorabbin (YMMB)/operations.md

```diff
@@ -0,0 +1,44 @@
+---
+  title: Operations
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier                         |
+| ------------------ | -------------- | ---------------- | ---------------------------------------- |
+| Moorabbin ADC  | Moorabbin Tower  | 118.100         | MB_TWR                        |
+| Moorabbin SMC   | Moorabbin Ground   | 134.250          | MB_GND                       |
+| Moorabbin ATIS        |                | 120.900          | YMMB_ATIS                               |
+
+ 
+In the absence of an SMC controller, ADC shall assume the SMC responsibilities.
+<figure markdown>
+![YSBK Maneuvering Area](img/ysbk_maneuvring_area.png){ width="500" }
```

Comment:
Fix this to YMMB not bankstown

Reply:
> Fix this to YMMB not bankstown

fixed. no idea where or what ds_store file is. todo conflict resolved (i think)